src/screens/PickList/index.tsx

```diff
@@ -445,13 +451,21 @@ const PickOrderItem = () => {
                       onChange={onChangeBin}
                       editable={false}
                     />
-                    <InputSpinner
-                      title={"Quantity to Pick"}
-                      setValue={quantityPickedChange}
+                    <InputBox
```

Comment:
Why there is input box instead of input spinner?

Reply:
because we change the component for InputSpinner and replace it with auto suggestion details

Reply:
@awalkowiak  These changes are already available. it's available in restructure branch.

Reply:
@hellochirag i think his point is that we don't want to switch back to InputBox. We want to use InputSpinner.

Reply:
https://user-images.githubusercontent.com/35322132/146552235-66db46f5-9f0f-4479-ae65-db2e4d66a6b2.mov

@awalkowiak @jmiranda when I started work on the issue there is an Inputbox and after merging other branch code [it's replaced with InputSpinner. Please check issue attached video. ](https://pihemr.atlassian.net/browse/OBKN-351)

Please let me know if we need to replace with components.

Reply:
https://user-images.githubusercontent.com/35322132/146554692-6689139b-3052-42ff-ae96-ce1c3ebc5670.mov

@jmiranda @awalkowiak resolve the OBKN-351 issue with **InputSpinner**  component as well, Please let me know which component we need to add.?